pocs/core.py

```diff
@@ -129,9 +129,8 @@ def initialize(self):
             self.say("Initializing the system! Woohoo!")
 
             try:
-                # Initialize the mount
-                self.logger.debug("Initializing mount")
-                self.observatory.mount.initialize()
+                self.logger.debug("Initializing observatory")
```

Comment:
Perhaps adjust the comment on line 72 to not claim the observatory is being created.
And in the ctor docstring on line 29, could you clarify the condition that the observatory instance is supposed to be in (esp. not yet initialized)?

And FWIW, it would be nice to continue with dependency injection for the Observatory class.

Reply:
Thanks, got those updated. Agree with continuing on with the Observatory dependency injection. Perhaps that should be after a patch release, which I'd still like to get done relatively soon.